packages/bot/src/components/summary.jsx

```diff
@@ -39,11 +39,11 @@ const Summary = ({
                 <div className='summary__tiles'>
                     <SummaryTile
                         title={localize('Total stake')}
-                        content={ Money({ amount: total_stake, currency }) }
+                        content={ <Money amount={total_stake} currency={currency} /> }
```

Comment:
@aaron-binary @Mahboobeh-binary had to change this cos `React.memo` doesn't work when calling this component as function